Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportApiHandler.cs

```diff
@@ -222,6 +223,30 @@ public async Task SendActionExecutionDataToCentralDBAsync(AccountReportAction ac
             }          
         }
 
+        public bool ExecutionIdValidation(Guid executionId)
+        {
+            RestRequest restRequest = (RestRequest)new RestRequest(EXECUTION_ID_VALIDATION + executionId, Method.GET);           
+            string message = string.Format("execution id : {0}", executionId);
+            try
+            {
+                IRestResponse response = restClient.Execute(restRequest);
+                if (response.IsSuccessful)
+                {
+                    Reporter.ToLog(eLogLevel.DEBUG, "Successfully validated execution id " + message);
```

Comment:
it's confusing, API will return 'true' if ExecutionID exist and 'false' if not?
this is ValidationAPI and not IsExist API- so need to return 'true' if validation passed (meaning not duplication) and 'false' if failed (meaning there is duplication)